app/src/main/java/com/door43/translationstudio/tasks/UpdateAppTask.java

```diff
@@ -149,6 +147,8 @@ private void updateBuildNumbers() {
      */
     private void upgradePre115() {
         AppContext.context().deleteDatabase(Library.DATABASE_NAME);
+        TargetTranslation[] targetTranslations = AppContext.getTranslator().getTargetTranslations();
+        TargetTranslationMigrator.migrateFromUSXtoUSFM(targetTranslations);
```

Comment:
@neutrinog - it looks like v4() was designed for migrating manifests.  Are we now extending this to update the files also?  Will try to get my head around this migration workflow.